itou/approvals/models.py

```diff
@@ -1794,7 +1794,7 @@ def find_for(self, user):
         if not filters:
             return self.none()
         or_filters = functools.reduce(operator.__or__, filters)
-        return self.filter(or_filters).order_by("-start_at")
+        return self.filter(or_filters).order_by("-start_at", "-number")
```

Comment:
Le numéro des Agréments PE n'est pas séquentiel comme l'est celui des PASS IAE donc la PK me semble plus appropriés (ou alors `-end_at` pour prendre privilégier celui qui se termine en dernier et donc le plus long ?) que le numéro qui lui est corrélé a l'agence émettrice et donc départementalisé/régionalisé.
```suggestion
        return self.filter(or_filters).order_by("-start_at", "pk")
```

Reply:
`-end_at` me paraît encore mieux, mais ne suffira pas pour la disambiguation dans le test (et en prod).
Je n’aime pas trop compter sur la PK, qui n’a pas de signification. Si on avait un UUID, l’ordre ne serait pas stable, car en ajoutant des entrées dans la table, on pourrait avoir un UUID qui serait “plus petit” que les existants, et changer le tri. Bon là on a un `IntegerField`, donc ça se discute, mais la PK reste un truc interne et difficile à partager.

C’est comme ça que je suis arrivé au `number`, qui est unique et partagé avec PE, donc dire “on prend celui qui a le numéro le plus grand” est facile à comprendre par le support et un éventuel interlocuteur de chez PE.

Donc je propose `.order_by("-start_at", "-end_at", "-number")` ?

Reply:
Ça concerne tellement peu de gens dorénavant que ne tergiversons pas :).
Mais en re-relisant je suit encore plus convaincu qu'on ne devrais pas trier par `number` (ni `pk` d'ailleurs) car le problème qu'on essaye de résoudre est synthétique et n'existe que dans nos tests, donc ça serait plutôt au test d'utiliser des `set()` plutôt qu'une `list()`.